person.rb

```diff
@@ -20,6 +20,14 @@ def correct_name
     @name
   end
 
+  def rentals
+    @rental
+  end
```

Comment:
- [ ] Definition of a new method is not what the previous reviewer meant. The previous reviewer meant you should add an empty rentals variable to your `initialize` constructor. See example below. Kindly remove this method as it is unnecessary.

  ```
  def initialize(age, name = 'unknown', parent_permission: true)
    super()
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
    @rentals = []
  end
  ```